### PR TITLE
Update translation

### PR DESCRIPTION
**Pull request by liijunwei**
+ http://www.catb.org/~esr/faqs/smart-questions.html
+ original: "Now that use of open source has become widespread, you can often get as good answers from other, more experienced users as from hackers. This is a Good Thing; users tend to be just a little bit more tolerant of the kind of failures newbies often have. Still, treating experienced users like hackers in the ways we recommend here will generally be the most effective way to get useful answers out of them, too."

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ __本指南不提供此專案的實際支援服務！__
 
 在[黑客](http://www.catb.org/~esr/faqs/hacker-howto.html)的世界裡，當你拋出一個技術問題時，最終是否能得到有用的回答，往往取決於你所提問和追問的方式。本指南將教你如何正確的提問以獲得你滿意的答案。
 
-不只是黑客，現在開放原始碼（Open Source）軟體已經相當盛行，你常常也可以由其他有經驗的使用者身上得到好答案，這是件**好事**；使用者比起黑客來，往往對那些新手常遇到的問題更寬容一些。然而，將有經驗的使用者視為黑客，並採用本指南所提的方法與他們溝通，同樣也是能從他們身上得到滿意回答的最有效方式。
+現在開源（Open Source）軟件已經相當盛行，您通常可以從其他更有經驗的用戶那裡獲得與黑客一樣好的答案，這是件**好事**；和黑客相比，用戶們往往對那些新手常遇到的問題更寬容一些。儘管如此，以我們在此推薦的方式對待這些有經驗的用戶通常也是從他們那裡獲得有用答案的最有效方式。
```
Comment (ryanhanwu) [inline README.md:79]: 在繁體中文仍是使用"軟體" 居多，可以煩請修正一下，感謝！

**Review by ryanhanwu — CHANGES_REQUESTED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (liijunwei) [reply README.md]: Fixed.

**Review by ryanhanwu — APPROVED** [review]
Much appreciated!

**Pull request merged**